std/math.d

```diff
@@ -5249,25 +5257,10 @@ bool isInfinity(X)(X x) @nogc @trusted pure nothrow
  */
 bool isIdentical(real x, real y) @trusted pure nothrow @nogc
 {
-    // We're doing a bitwise comparison so the endianness is irrelevant.
-    long*   pxs = cast(long *)&x;
-    long*   pys = cast(long *)&y;
     alias F = floatTraits!(real);
-    static if (F.realFormat == RealFormat.ieeeDouble)
-    {
-        return pxs[0] == pys[0];
-    }
-    else static if (F.realFormat == RealFormat.ieeeQuadruple
-                 || F.realFormat == RealFormat.ibmExtended)
-    {
-        return pxs[0] == pys[0] && pxs[1] == pys[1];
-    }
-    else
-    {
-        ushort* pxe = cast(ushort *)&x;
-        ushort* pye = cast(ushort *)&y;
-        return pxe[4] == pye[4] && pxs[0] == pys[0];
-    }
+    F.Layout repX = { number : x }, repY = { number : y };
+
+    return repX.integral == repY.integral;
```

Comment:
This would be turned into a ``memcmp`` for ``ieeeQuadruple`` and ``ibmExtended``, and goodness knows what for ``ieeeExtended``.  For efficiency, I'd recommend keeping the previous behaviour and inline the bit comparison.